packages/react-components/react-text/src/stories/TextTypography.stories.tsx

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import { Body, Caption, Display, Headline, LargeTitle, Title1, Title2, Title3, Subheadline } from '../index';
+import { Body1, Caption1, Display, Subtitle1, LargeTitle, Title1, Title2, Title3, Subtitle2, Caption2 } from '../index';
```

Comment:
_nit:_ Keep the alphabetical order

```suggestion
import { Body1, Caption1, Caption2, Display, LargeTitle, Title1, Title2, Title3, Subtitle1, Subtitle2 } from '../index';
```